util/scale_run.sh

```diff
@@ -0,0 +1,230 @@
+BEAKER_PE_DIR=$BEAKER_PE_DIR
+BEAKER_PE_VER=$BEAKER_PE_VER
+PUPPET_GATLING_SCALE_BASE_INSTANCES=$PUPPET_GATLING_SCALE_BASE_INSTANCES
+PUPPET_GATLING_SCALE_ITERATIONS=$PUPPET_GATLING_SCALE_ITERATIONS
+PUPPET_GATLING_SCALE_INCREMENT=$PUPPET_GATLING_SCALE_INCREMENT
+PUPPET_GATLING_SCALE_SCENARIO=$PUPPET_GATLING_SCALE_SCENARIO
+ABS_AWS_MOM_SIZE=$ABS_AWS_MOM_SIZE
+============================================"
+}
+
+# Link scale results to nginx content location
+function link_results () {
+    # test name is arg1, pe_ver is arg2
+    source=$HOME/gplt/$1/gatling-puppet-load-test/results/scale
+    dest=/usr/share/nginx/html/gplt/scale/$1/$2
+
+    sudo mkdir -p "${dest}"
+    sudo ln -s "${source}" "${dest}"
+}
+
+# Prepare checkout of puppet-gatling-load-test
+function prep_gplt () {
+    cd "$HOME/gplt" || exit 1
+    mkdir "$1"
+    cd "$1" || exit 1
+    git clone git@github.com:puppetlabs/gatling-puppet-load-test
+    cd gatling-puppet-load-test || exit 1
+    bundle install --path vendor/bundle
+}
+
+
+##########
+# Argument checking
+##########
+
+ARGS=()
+while [ $# -gt 0 ]
+do
+    unset OPTIND
+    unset OPTARG
+    while getopts hd  options
+    do
+    case $options in
+        h)  usage
+            exit 1
+            ;;
+        d)  DEBUG=true
+            ;;
+        esac
+   done
+   shift $((OPTIND-1))
+   ARGS+=($1)
+   shift
+done
+
+# Ensure PE_VERSION is provided
+if [ "${#ARGS[@]}" -le "0" ]; then
+    echo "PE_VERSION not provided"
+    echo
+    usage
+    exit 1
+fi
+PE_VERSION=${ARGS[0]}
+
+# Ensure PE_VERSION is semver
+# e.g. 2019.1.1-rc0-207-g4e47830
+# Regex from: https://regexr.com/39s32
+if ! [[ $PE_VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+((-[0-9a-zA-Z-]+)*)?$ ]]; then
+    echo "Supplied PE_VERSION is malformed"
+    echo
+    usage
+    exit 1
+fi
+
+
+##########
+# Main
+##########
+
+tune=false
+# Global settings
+export BEAKER_INSTALL_TYPE=pe
+export BEAKER_PE_DIR=http://enterprise.delivery.puppetlabs.net/archives/releases/2019.1.0
+export BEAKER_PE_VER=$PE_VERSION
+export PUPPET_GATLING_SCALE_TUNE=$tune
+# export PUPPET_GATLING_SCALE_TUNE_FORCE=true
+
+
+mkdir "$HOME/gplt"
+
+echo "========================================================================"
+echo "Testing Ref Arch 1: Monolithic deploy for 10 or fewer nodes"
+
+    export PUPPET_GATLING_SCALE_BASE_INSTANCES=100
+    export PUPPET_GATLING_SCALE_ITERATIONS=1
+    export PUPPET_GATLING_SCALE_SCENARIO="Scale.json"
+    export ABS_AWS_MOM_SIZE="m5.large"
+
+    run_type="cold"
+    ec2=$ABS_AWS_MOM_SIZE
+    test="slv-414-ref1-$ec2"
+    cmd="bundle exec rake autoscale_$run_type > \"$test-$run_type.log\""
+    if [ -z $DEBUG ]; then
+        prep_gplt "$test"
+        (bundle exec rake autoscale_$run_type > "$test-$run_type.log") &
+    else
+        (echo_env) &
+    fi
+
+
+echo "========================================================================"
+echo "Testing Ref Arch 2: Monolithic deploy for up to 4,000 nodes"
```

Comment:
Replace Mono with Standard.  Remove hardcoded node assumption since that is what we are testing...